samples/samples/requirements.txt

```diff
@@ -1,2 +1,2 @@
-google-cloud-spanner==3.48.0
+google-cloud-spanner==3.49.0
```

Comment:
This should not be done manually, this was only for testing.

Reply:
Changes removed